goc/config-osg/libexec/osgstorage-auth.conf

```diff
@@ -1,11 +1,19 @@
+# This file is sourced by cvmfs config files but not parsed by cvmfs
+# In order for cvmfs to export its variables, the sourcing config file has
+#  to first set the following variables:
+CVMFS_AUTHZ_OSG_BASE_DIR=
+CVMFS_AUTHZ_OSG_WN_CLIENT=
+CVMFS_AUTHZ_HELPER=
+X509_CERT_DIR=
+
 distroversion()
 {
     # systemd-based systems have /etc/os-release
     # Prefer /etc/os-release because lsb_release isn't always installed
     #   on systems that have /etc/os-release
     if [ -f /etc/os-release ]; then
-        # Can't use equal sign in sed script, it confuses cvmfs variable parser
-        sed -n 's/"//g;s/^VERSION_ID[^A-Z]//p' /etc/os-release
+        # If cvmfs parsed this file, couldn't use equal sign in sed script
```

Comment:
This comment seems superfluous if the config file isn't being parsed by cvmfs, no?

Reply:
Yeah, but it used to be parsed by cvmfs and maybe it will again one day so I wanted to note the possibility somewhere.

Reply:
Could you fix the grammar then? It was rather confusing

```
# If cvmfs parses this file, you can't use an equal sign in the sed script
```

Reply:
I could do it in a separate pull request, but this is the version currently released so I want to tag it as it is first.

Reply:
Alright, submit the PR and link it here then I'll approve this :)

Reply:
Ok, it is #78. That PR includes this one, so the diff is easier to see in commit 6b0beca.